docs/next/en-US/deploying/docker.md

```diff
@@ -0,0 +1,258 @@
+# Docker Deployment
```

Comment:
Could you please apply the same changes I requested for the v0.3.x version here?